stock_available_immediately_exclude_location/models/product_product.py

```diff
@@ -9,34 +9,60 @@ class ProductProduct(models.Model):
 
     _inherit = "product.product"
 
-    def _compute_available_quantities_dict(self):
+    def _compute_quantities_dict(
+        self, lot_id, owner_id, package_id, from_date=False, to_date=False
+    ):
         """
-        change the way immediately_usable_qty is computed by deducing the quants
+        Re-compute the the `free to use` qty by deducting the quants
         in excluded locations
         """
```

Comment:
Better to not override the original docstring of the method and to write this as a comment instead